src/main/pages/overview/infra-support.adoc

```diff
@@ -10,74 +10,65 @@ folder: overview
+=== Available features for different infrastructures
 
+[%autowidth]
 |===
-| Feature | *Docker* | *OpenShift* | *Kubernetes* |
-|root access | yes | yes (See: link:openshift-config.html#enable-ssh-and-sudo[Configuration]) | yes |
-|https | no | yes (See: link:openshift-config.html#https-mode[Configuration]) | yes |
-|scalability | no | yes (See: link:openshift-config.html#scalability[Configuration]) | yes |
-|priviliged containers | yes | yes (configurable in https://docs.openshift.com/container-platform/3.6/admin_guide/manage_scc.html#grant-access-to-the-privileged-scc[OpenShift]) | yes |
-|health checks | no | yes | yes |
-|persistent preview URLs | no | yes | yes |
-|installers | yes | yes (some installers may require link:openshift-config.html#enable-ssh-and-sudo[sudo access]) | yes |
-|file system permissions | not limited | limited to directories owned by root link:openshift-config.html#filesystem-permissions[group] | not limited |
+| Feature | *Docker* | *OpenShift* | *Kubernetes* 
+|root access | yes | yes (See: link:openshift-config.html#enable-ssh-and-sudo[Configuration]) | yes 
+|https | no | yes (See: link:openshift-config.html#https-mode[Configuration]) | yes 
+|scalability | no | yes (See: link:openshift-config.html#scalability[Configuration]) | yes 
+|privileged containers | yes | yes (configurable in https://docs.openshift.com/container-platform/3.6/admin_guide/manage_scc.html#grant-access-to-the-privileged-scc[OpenShift]) | yes 
+|health checks | no | yes | yes 
+|persistent preview URLs | no | yes | yes 
+|installers | yes | yes (some installers may require link:openshift-config.html#enable-ssh-and-sudo[sudo access]) | yes 
+|file system permissions | not limited | limited to directories owned by root link:openshift-config.html#filesystem-permissions[group] | not limited 
 |===
 
-[id="running-che-on-docker"]
-== Running Che on Docker
-
-Che on Docker isn’t scalable, i.e. one cannot add more nodes to run workspaces. Of course, it is possible to use a fairly large instance with 8+ CPUs and 32+ RAM, however at some point a great number of running containers (with heavy processes running in them) can make the node unresponsive. This will both affect workspace master and all running workspaces. At the same time, Che on Docker is flexible and easily configurable for an ordinary user.
-
-*Root Access*
+[id="docker-versus-openshift-for-running-che"]
+== Docker versus OpenShift for running Che
 
-With Che on Docker, users can have root access in workspace containers. This means you can run system services and install software in runtime.
+Che on Docker is not scalable because you can not add more nodes to run workspaces. You can use a large instance with 8 CPUs or more and 32+ RAM.  However, a potential problem is that the instance can be unresponsive when it has a large number of running containers with heavy processes running in them. This affects the workspace master and all running workspaces.
```

Comment:
* cannot
* 32+ of what? GB?
* don't use 'however'

Reply:
I will add GB for now but will confirm with @eivantsov. Eugene is it 32 GB of RAM?

Reply:
Chatted with Eugene on Mattersmost.. Decided to remove "You can use a large instance with 8 CPUs or 32+ RAM"   and replace with "Even if you have a node with lots of resources, you can face problems with many containers residing on one node.